File: KotlinTemplates/app/src/main/java/com/kotlintemplates/DInjection/View/View/ResultModel.java
package com.kotlintemplates.DInjection.View.View;

import com.google.gson.annotations.SerializedName;

public class ResultModel {
    @SerializedName("id")
    private  int id = 0;
    @SerializedName("body")
    private String body;
    @SerializedName("title")
    private String title;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: KotlinTemplates/app/src/main/java/com/kotlintemplates/RoomRetrofitList/Repository/ResultModel.kt
package com.kotlintemplates.RoomRetrofitList.Repository

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import android.support.annotation.NonNull
import com.google.gson.annotations.SerializedName

@Entity(tableName = "post_info")
class ResultModel {
    @PrimaryKey
    @NonNull
    @ColumnInfo(name = "id")
    @SerializedName("id")
    private var id: Int = 0
    @NonNull
    @ColumnInfo(name = "body")
    @SerializedName("body")
    private var body: String? = null
    @NonNull
    @ColumnInfo(name = "title")
    @SerializedName("title")
    private var title: String? = null


    fun getId(): Int {
        return id
    }

    fun setId(id: Int) {
        this.id = id
    }

    fun getBody(): String? {
        return body
    }

    fun setBody(body: String) {
        this.body = body
    }

    fun getTitle(): String? {
        return title
    }

    fun setTitle(title: String) {
        this.title = title
    }



    override fun toString(): String {
        return "ClassPojo [id = $id, body = $body, title = $title]"
    }
}